src/vs/editor/browser/widget/multiDiffEditor/multiDiffEditorWidgetImpl.ts

```diff
@@ -195,9 +195,9 @@ export class MultiDiffEditorWidgetImpl extends Disposable {
 		const viewItems = this._viewItems.get();
 		let searchCallback: (item: VirtualizedViewItem) => boolean;
 		if ('original' in resource) {
-			searchCallback = (item) => item.viewModel.originalUri?.toString() === resource.original.toString();
+			searchCallback = (item) => item.viewModel.originalUri?.path === resource.original?.path;
```

Comment:
This is not a good change. `reveal` shouldn't ignore the protocol scheme. Can you fix this at the caller?